Sources/SwerverCore/Action/OptionsAction.swift

```diff
@@ -21,9 +21,12 @@ public class OptionsAction: HttpAction {
     
     private func getAllowedMethods() -> String {
         var allowedMethodList: [String] = []
-        for (method, _) in routes.fetchRoute(url: route) {
-            allowedMethodList.append(method.rawValue)
+        if let actions = routes.fetchRoute(url: route)?.actions {
```

Comment:
Determining the available actions for a given route feels like the responsibility of the `Routes` class.

I would consider refactoring to introduce something like this:
```swift
class Routes {
    func allowedMethods(url: URL) -> [HttpMethod] {
    }
}
```

Reply:
will look into that